cas-server-webapp-support/src/main/java/org/jasig/cas/web/support/InspektrThrottledSubmissionByIpAddressAndUsernameHandlerInterceptorAdapter.java

```diff
@@ -148,6 +148,6 @@ public final void setAuthenticationFailureCode(final String authenticationFailur
      */
     protected String constructUsername(final HttpServletRequest request, final String usernameParameter) {
         final String username = request.getParameter(usernameParameter);
-        return "[username: " + (username != null ? username : "") + ']';
+        return (username != null ? username : "unknown") + "+password";
```

Comment:
The constructUsername() method still need to be modified, because the database is storing "username" not "[username: username]";